conda-recipe-openblas/meta.yaml

```diff
@@ -10,48 +10,30 @@ build:
   string: py{{ environ.get('PY_VER').replace('.', '') }}{{ environ.get('GIT_DESCRIBE_HASH', '') }}_obl_{{ environ.get('GIT_DESCRIBE_NUMBER', 0) }}
 
 requirements:
-  # TODO: pip possibly required in only *one* of build and host, but which?
   build:
-    - python {{ PY_VER }}
-    - numpy
-    - openblas >=0.3.0
-    - {{ compiler('c') }}       # [not osx]
-    - gcc                       # [osx]
-    - {{ compiler('fortran') }} # [linux]
-    - scikit-build >=0.8.0
+    - {{ compiler('c') }}       
+    - {{ compiler('fortran') }}   # [unix]
+    - flang                       # [win]
 
   host:
-    - python {{ PY_VER }}
-    - pip
-    - flang                     # [win]
     - numpy
-    - openblas >=0.3.0
-    - libgfortran-ng # [not-win]
-    - libgcc-ng      # [linux]
-    - scikit-build >=0.8.0
-   # on Windows, this relies on having visual studio CE 2015
-   # this link needed quite some searching, please do not delete!
-   # https://go.microsoft.com/fwlink/?LinkId=532606&clcid=0x409
-   
+    - libopenblas
+    - python
+    - scikit-build
+    - pip
+    
   run:
     - python {{ PY_VER }}
     - {{ pin_compatible('numpy') }}
```

Comment:
need to add - libopenblas here

Reply:
You should be able to write to this branch yourself?

Reply:
I think I need write access to python-control/Slycot for that...

Would the single failure on Linux with Python 3.5 be fixed with a 
export LD_LIBRARY_PATH="$HOME/miniconda/envs/test-environment/lib:";

OSX  needs 10.9 SDK?

# on OSX, the SDK for 10.9 is currently needed
# download the 10.9 sdk from https://github.com/phracker/MacOSX-SDKs/releases
# unpack and set environment variable CONDA_BUILD_SYSROOT to that location

Reply:
> I think I need write access to python-control/Slycot for that...

Done.

Reply:
the single failure was fixed by adding the BLA_VENDOR variable.

I don't understand the MacOS X setup. Travis has settings for different xcode installations. See also https://github.com/bnavigator/Slycot/tree/work-on-build-plus-travis

Reply:
Continue discussion in #92 and #93